conf/modules/air_data.xml

```diff
@@ -36,6 +36,7 @@
   </header>
   <init fun="air_data_init()"/>
   <periodic fun="air_data_periodic()" freq="10"/>
+  <datalink message="WIND_INFO" fun="air_data_parse_WIND_INFO(dev,trans,buf)"/>
```

Comment:
Since you set `dev` and `trans` to unused, you can also just remove them here instead.

Reply:
they are used only when `WIND_INFO_RET` is defined, so I need to keep them with the unused flag to avoid warnings in default case